spec/lib/trln_argon/solr_document_spec.rb

```diff
@@ -864,83 +864,51 @@ class SolrDocumentTestClass
   ########################
   # Citation
   ########################
-  describe 'citable?' do
-    context 'document does not have an oclc number' do
-      let(:document) do
-        SolrDocumentTestClass.new(
-          id: 'TRLN12345'
-        )
-      end
-
-      it 'returns false' do
-        expect(document.citable?).to be false
-      end
+  context 'when document has all required fields for citations' do
```

Comment:
Great to see tests for this feature. I found the "has all required fields" confusing though, since that implies that if a document is missing one of these fields then it won't get a citation. Can this be made clearer? If really every item will get some kind of citation now, can you include a test or two that demonstrate what it looks like when there's minimal metadata present?